Dockerfile

```diff
@@ -10,4 +10,17 @@ RUN dotnet publish OpenImis.RestApi/OpenImis.RestApi.csproj -c $BUILD-FLAVOUR -o
 FROM mcr.microsoft.com/dotnet/core/aspnet:2.1
 WORKDIR /app
 COPY --from=build-env /app/OpenImis.RestApi/out .
-ENTRYPOINT ["dotnet", "OpenImis.RestApi.dll"]
+
+ENV DB_HOST=Server
```

Comment:
Should this be taken form .env file?

Reply:
No, here I just set the default value, we can remove the value or even the line but I think it is good for documentation purpose